docs/data/joy/components/tabs/tabs.md

```diff
@@ -24,41 +24,34 @@ Joy UI provides four tabs-related components:
 
 {{"demo": "TabsUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
-## Component
-
-After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
+## Basics
 
 ```jsx
 import Tabs from '@mui/joy/Tabs';
 import TabList from '@mui/joy/TabList';
 import Tab from '@mui/joy/Tab';
-
-export default function MyApp() {
-  return (
-    <Tabs defaultValue={1}>
-      <TabList>
-        <Tab value={1}>Tab A</Tab>
-        <Tab value={2}>Tab B</Tab>
-        <Tab value={3}>Tab C</Tab>
-      </TabList>
-    </Tabs>
-  );
-}
 ```
 
-### Basic usage
+The Joy UI set of Tabs components follows the [WAI ARIA design pattern guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).
 
-The tabs structure follows [WAI ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).
-To target the initially selected tab, specify the `value` prop to the `TabPanel` and use `Tabs`'s `defaultValue`.
+Use the `value` prop on the Tab Panel and the `defaultValue` prop on the Tabs component to target the selected tab.
 
 {{"demo": "TabsBasic.js"}}
 
+### Disabled tab
+
+Use the `disabled` prop to disable interaction and focus.
+
+{{"demo": "TabDisabled.js"}}
+
+## Customization
+
 ### Variants
 
-Both `TabList` and `Tab` accept [global variant](/joy-ui/main-features/global-variants/) values, so you can mix and match to get the desired result.
+The Tab List and Tab components accept the global `variant` prop values.
```

Comment:
Since "Tab components" could refer to `<Tab />` _or_ the whole collection of components, I'd style it this way for clarity:

```suggestion
The `<TabList />` and `<Tab />` components accept the global `variant` prop values.
```